openapi_test.go

```diff
@@ -39,6 +39,16 @@ type MyOutputStruct struct {
 	Quantity int    `json:"quantity"`
 }
 
+type MyOutputStructDetails struct {
+	IngressDate string `json:"in_date" xml:"date,attr"`
+}
+
+type MyOutputStructWithXmlAttribute struct {
```

Comment:
I think we should add a test for a struct with just xml attributes. As well as one with `json:"-"`